media/css/mozorg/home/home-b.less

```diff
@@ -0,0 +1,1160 @@
+
+    h2 {
+        .font-size(24px);
+        padding-top: 40px;
+
+        &:before {
+            .at2x('/media/img/firefox/common/firefox-logo.png');
+            .background-size(100%);
+            background-repeat: no-repeat;
+            content: '';
+            display: block;
+            height: 84px;
+            margin: 0 auto 10px;
+            width: 80px;
+        }
+
+        @media @mq-phone-wide {
+            .font-size(36px);
+
+            &:before {
+                height: 142px;
+                width: 136px;
+            }
+        }
+
+        @media @mq-highres {
+            &:before {
+                .background-size(100%);
+            }
+        }
+    }
+
+    .horizon {
+        .horizon-gradient;
+
+        .content {
+            padding-bottom: 270px;
+        }
+    }
+
+    .stars {
+        background: url('/media/img/firefox/horizon/stars.svg') center 60px no-repeat;
+    }
+
+    .foreground .content {
+        padding-bottom: 60px;
+
+        p {
+            margin-bottom: 40px;
+        }
+
+        a,
+        a:hover {
+            color: #fff;
+        }
+
+        @media @mq-tablet {
+            .font-size(24px);
+        }
+    }
+
+    .forest-container {
+        height: 483px;
+        margin-top: -254px;
+        position: relative;
+    }
+
+    .mountains-container {
+        height: 483px;
+        left: 50%;
+        // offset assets to get the fox in the viewport
+        margin-left: -1046px;
+        position: absolute;
+        top: 0;
+        width: 1770px;
+
+        .layer {
+            left: 0;
+            position: absolute;
+            width: 1770px;
+        }
+
+        @media @mq-tablet {
+            // re-center assets
+            margin-left: -885px;
+        }
+    }
+
+    .sun {
+        height: 318px;
+        left: 50%;
+        margin-left: -161px;
+        position: absolute;
+        top: 30px;
+        width: 322px;
+    }
+
+    .mountains {
+        height: 482px;
+        top: 0;
+    }
+
+    .foreground-shape {
+        .at2x('/media/img/firefox/horizon/foreground.png', 1770px, 254px);
+        background-position: top left;
+        background-repeat: no-repeat;
+        bottom: 0;
+        height: 254px;
+    }
+
+    .trees-left {
+        bottom: 70px;
+        height: 602px;
+        left: -375px;
+        position: absolute;
+        width: 909px;
+    }
+
+    .trees-right {
+        bottom: 120px;
+        height: 550px;
+        position: absolute;
+        right: -420px;
+        width: 922px;
+    }
+
+    .fox {
+        bottom: 39px;
+        height: 254px;
+        position: absolute;
+        right: 140px;
+        width: 522px;
+    }
+
+    #fx-download-link {
+        display: none;
+
+        @media @mq-tablet {
+            display: inline-block;
+        }
+    }
+
+    #fxmobile-download-buttons {
+        li {
+            margin-bottom: 12px;
+        }
+
+        @media @mq-tablet {
+            display: none;
```

Comment:
Would it be better to hide/show these based on platform rather than viewport size?